docs/manual/constant-value-checker.tex

```diff
@@ -251,6 +251,39 @@ \section{Warnings\label{value-checker-warnings}}
 
 \end{itemize}
 \end{sloppypar}
+
+
+\section{Overflow\label{value-checker-overflow}}
+
+The Constant Value Checker normally takes Java's normal overflow rules into account when
+computing the possible values of expressions.
+The \code{-AignoreRangeOverflow} command-line option
+makes it ignore the possibility of overflow
+for range annotations (\<@IntRange>, for instance), making them more precise.
+You can supply the \code{-AignoreRangeOverflow} command-line option
+when you are confident that no overflow will occur in the program (for
```

Comment:
Even if the Index Checker to be built on the assumption that it doesn't have to check overflow, that's different from it being built on the assumption that overflow does not occur.  You have also changed the discussion from being about the Value Checker to being about the Index Checker -- which is this really about?

I think you can clearly explain what is going on, and I would like you to do so even if it isn't easy.  So far, the patches and the discussion have seemed to be about putting in hacks to prevent warnings, without necessarily understanding the implications of those hacks.  Let's make the design proceed from an understanding of the issues, rather than making changes and then attempting to triangulate back to a design that would justify those changes.

Thanks.